DataMgr/FileMgr/FileMgr.cpp

```diff
@@ -948,7 +948,7 @@ void FileMgr::getChunkMetadataVecForKeyPrefix(
                           // read here for chunk
   auto chunkIt = chunkIndex_.lower_bound(keyPrefix);
   if (chunkIt == chunkIndex_.end()) {
-    CHECK(false);
+    return; // throw?
```

Comment:
With this check usual select queries (not foreign storage) for empty tables not working on my computer.
Also, with this check, I get errors in initdb. 
Looks like code is not ready yet to throw in this place :)